packages/nextjs/app/api/grants/review/route.tsx

```diff
@@ -1,9 +1,12 @@
-import { NextResponse } from "next/server";
+import { revalidatePath } from "next/cache";
+import { NextRequest, NextResponse } from "next/server";
 import { getAllGrantsForReview } from "~~/services/database/grants";
 
-export async function GET() {
+export async function GET(request: NextRequest) {
   try {
+    const path = request.nextUrl.searchParams.get("path");
     const grants = await getAllGrantsForReview();
+    if (path) revalidatePath(path);
     return NextResponse.json({ data: grants });
```

Comment:
This is a bit confusing but works!

I tried the 4th option from the documentation... and it works too!

![image](https://github.com/BuidlGuidl/grants.buidlguidl.com/assets/2486142/e76f9e69-1519-45a1-bdce-23b486e82112)

https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic

```ts
export const dynamic = "force-dynamic";
```

This seems more intuitive and also more aligned with the `revalidate` on the frontpage.

Reply:
What do you think? Am I missing something?

Reply:
Yup yup, I actually had this thought in mind that instead of (unnecessary)accessing `request` and making it dynamic, we could [`export const revalidate = 0`](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate)

```ts
export const dynamic = "force-dynamic";
``

Yup this seems more intuitive than `revalidate = 0` updated it at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/60/commits/ce1e6a4b6fc52a516dbe54dae2f73f9841ad4849, Tysm 🙌